cli/dcoscli/package/main.py

```diff
@@ -5,7 +5,8 @@
     dcos package --info
     dcos package describe [--app --options=<file> --cli] <package_name>
     dcos package install [--cli | [--app --app-id=<app_id>]]
-                         [--options=<file> --yes] <package_name>
+                         [--pkg-version=<pkg_version>] [--options=<file> --yes]
```

Comment:
Hmmm, yea, I prefer "version", and it's possible technically, but we'd have to differentiate in the help message, which is confusing.

I like Jose's suggestion to use "package version" and "package revision".  So can we call this `--package-version`?  Sorry for the back and forth